conanfile.py

```diff
@@ -11,27 +11,14 @@ class RapidJSONAdapterConan(ConanFile):
     license = "MIT"
     generators = "cmake_find_package"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"gtest": ["1.7.0", "1.8.1", "1.10.0"]}
-    default_options = "gtest=1.10.0"
-    exports_sources = "*", "!build-*", "!*.yml", "!*.md", "!*.in", "!ci", "!.gitattributes", "!.gitignore", "!LICENSE"
-
-    def configure(self):
-        self.options["JSONAdapterTestUtilities"].gtest = self.options.gtest
+    exports_sources = "*", "!build/*", "!*.yml", "!*.md", "!*.in", "!ci", "!.gitattributes", "!.gitignore", "!LICENSE"
 
     def requirements(self):
-        self.requires("JSONAdapterInterface/1.1.5@systelab/stable")
-        self.requires("rapidjson/1.1.0.1@systelab/stable")
+        self.requires("JSONAdapterInterface/1.1.6@systelab/stable")
+        self.requires("rapidjson/1.1.0.2@systelab/stable")
 
     def build_requirements(self):
-        self.build_requires("JSONAdapterTestUtilities/1.1.5@systelab/stable")
-        if self.options.gtest == "1.7.0":
-            self.build_requires("gtest/1.7.0@systelab/stable")
-        elif self.options.gtest == "1.8.1":
-            self.build_requires("gtest/1.8.1")
-        elif self.options.gtest == "1.10.0":
-            self.build_requires("gtest/1.10.0#0c895f60b461f8fee0da53a84d659131")
-        else:
-            self.build_requires(f"gtest/{self.options.gtest}")
+        self.build_requires("JSONAdapterTestUtilities/1.1.6@systelab/stable")
```

Comment:
Are you not specifying gtest here or as private dependency because JSONAdapterTestUtilities already does it?

Reply:
It is already a dependency of JSONAdapterTestUtilites so it is already present in the build_requirements dependency tree. I'll add it again.

Reply:
Added it back again for readability